src/Account.java

```diff
@@ -12,18 +12,38 @@
 
     // Associations
     private HashMap<String, Payment> payments;
-    private HashMap<String, Order> orders;
+    private ArrayList<Order> orders;
```

Comment:
why the change?

Reply:
because we dont need to return a specific order, only the last one and it will be easier to return the last index in the arraylist